server/package.json

```diff
@@ -11,12 +11,12 @@
     "seed": "tsx prisma/seed"
   },
   "scripts": {
-    "test": "node --test test/**/*.test.js",
-    "start": "fastify start -l info app.js",
+    "db:reset": "npx prisma migrate reset --force",
     "dev": "npx prisma migrate dev && fastify start -w -l info -P app.js",
-    "migrate": "docker-compose exec server sh -c 'cd server && npx prisma migrate $@' sh",
-    "db": "docker-compose exec server sh -c 'cd server && npx prisma db $@' sh",
-    "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0"
+    "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
+    "start": "fastify start -l info app.js",
+    "test": "node --test $(find test -name '*.test.js')",
+    "test:db:reset": "DATABASE_URL=${DATABASE_URL}_test npx prisma migrate reset --force"
```

Comment:
I've changed the server scripts so that they don't reference docker. It should be easy enough for anyone who needs to run them to exec into their containers themselves. And for those who may not be using docker, or in our final deployments, they can still be used.

I've added a script to set up/reset a separate test db.